src/queueManager.js

```diff
@@ -186,6 +186,7 @@ export default class QueueManager {
           if (response.KVAPID) {
             Utils.log.debug(`kaios vapid recieved: ${response.KVAPID}`);
             Device.setVAPID(response.KVAPID);
+            Device.setVAPIDState(true);
```

Comment:
Should set VAPIDState to true on response of "App Launched" event. If an account doesn't have VAPID set on the dashboard VAPIDState will never be true causing the setTimeout in `_registerCTNotifications` to execute continuously in the background.

Reply:
@royson-ct Got it, will make changes accordingly

Reply:
@royson-ct How do I check if a certain response is only for the "App Launched" event? Should we set a max number of retries for the `_registerCTNotifications` method to be called in case VAPID is never set on the dashboard?